.github/contributing.md

```diff
@@ -16,13 +16,13 @@ You can contribute in several ways:
   : Report bugs at <https://github.com/munich-quantum-toolkit/bench/issues> using the _🐛 Bug report_ issue template. Please make sure to fill out all relevant information in the respective issue form.
 
 - 🐛 Fix Bugs
-  : Look through the [GitHub Issues] for bugs. Anything tagged with "bug" is open to whoever wants to try and fix it.
+  : Look through the [GitHub Issues](https://github.com/munich-quantum-toolkit/bench/issues) for bugs. Anything tagged with "bug" is open to whoever wants to try and fix it.
```

Comment:
```suggestion
  : Look through the [GitHub Issues] for bugs. Anything tagged with "bug" is open to whoever wants to try and fix it.
```
This is not really necessary as `[GitHub Issues]` is defined at the very bottom.